mantaray/received.py

```diff
@@ -228,6 +235,14 @@ def _handle_quit(server_view: views.ServerView, nick: str, args: list[str]) -> N
             view.userlist.remove_user(nick)
 
 
+def _handle_away(server_view: views.ServerView, nick: str, args: list[str]) -> None:
+    for view in _get_views_relevant_for_nick(server_view, nick):
+        if not args:
+            view.userlist.treeview.item(nick, tag=[])
+        else:
+            view.userlist.treeview.item(nick, tag=["away"])
```

Comment:
Can you add a new method to the userlist,  `set_away(nick: str, away: bool)` or something like that? Code outside `class _UserList` shouldn't need to access the treeview directly.